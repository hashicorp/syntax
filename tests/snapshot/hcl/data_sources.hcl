# Copyright (c) HashiCorp, Inc.

data "aws_ami" "example" {
  most_recent = true

  owners = ["self"]
  tags = {
    Name   = "app-server"
    Tested = "true"
  }
}
