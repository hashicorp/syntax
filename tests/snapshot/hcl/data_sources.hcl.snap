>data "aws_ami" "example" {
#^^^^ source.hcl meta.block.hcl entity.name.type.hcl
#    ^ source.hcl meta.block.hcl
#     ^^^^^^^^^ source.hcl meta.block.hcl variable.other.enummember.hcl
#              ^ source.hcl meta.block.hcl
#               ^^^^^^^^^ source.hcl meta.block.hcl variable.other.enummember.hcl
#                        ^ source.hcl meta.block.hcl
#                         ^ source.hcl meta.block.hcl punctuation.section.block.begin.hcl
>  most_recent = true
#^^ source.hcl meta.block.hcl
#  ^^^^^^^^^^^ source.hcl meta.block.hcl variable.declaration.hcl variable.other.readwrite.hcl
#             ^ source.hcl meta.block.hcl variable.declaration.hcl
#              ^ source.hcl meta.block.hcl variable.declaration.hcl keyword.operator.assignment.hcl
#               ^ source.hcl meta.block.hcl variable.declaration.hcl
#                ^^^^ source.hcl meta.block.hcl constant.language.hcl
>
>  owners = ["self"]
#^^ source.hcl meta.block.hcl
#  ^^^^^^ source.hcl meta.block.hcl variable.declaration.hcl variable.other.readwrite.hcl
#        ^ source.hcl meta.block.hcl variable.declaration.hcl
#         ^ source.hcl meta.block.hcl variable.declaration.hcl keyword.operator.assignment.hcl
#          ^ source.hcl meta.block.hcl variable.declaration.hcl
#           ^ source.hcl meta.block.hcl punctuation.section.brackets.begin.hcl
#            ^ source.hcl meta.block.hcl string.quoted.double.hcl punctuation.definition.string.begin.hcl
#             ^^^^ source.hcl meta.block.hcl string.quoted.double.hcl
#                 ^ source.hcl meta.block.hcl string.quoted.double.hcl punctuation.definition.string.end.hcl
#                  ^ source.hcl meta.block.hcl punctuation.section.brackets.end.hcl
>  tags = {
#^^ source.hcl meta.block.hcl
#  ^^^^ source.hcl meta.block.hcl variable.declaration.hcl variable.other.readwrite.hcl
#      ^ source.hcl meta.block.hcl variable.declaration.hcl
#       ^ source.hcl meta.block.hcl variable.declaration.hcl keyword.operator.assignment.hcl
#        ^ source.hcl meta.block.hcl variable.declaration.hcl
#         ^ source.hcl meta.block.hcl meta.braces.hcl punctuation.section.braces.begin.hcl
>    Name   = "app-server"
#^^^^ source.hcl meta.block.hcl meta.braces.hcl
#    ^^^^ source.hcl meta.block.hcl meta.braces.hcl meta.mapping.key.hcl variable.other.readwrite.hcl
#        ^^^ source.hcl meta.block.hcl meta.braces.hcl
#           ^ source.hcl meta.block.hcl meta.braces.hcl keyword.operator.assignment.hcl
#            ^ source.hcl meta.block.hcl meta.braces.hcl
#             ^ source.hcl meta.block.hcl meta.braces.hcl string.quoted.double.hcl punctuation.definition.string.begin.hcl
#              ^^^^^^^^^^ source.hcl meta.block.hcl meta.braces.hcl string.quoted.double.hcl
#                        ^ source.hcl meta.block.hcl meta.braces.hcl string.quoted.double.hcl punctuation.definition.string.end.hcl
>    Tested = "true"
#^^^^ source.hcl meta.block.hcl meta.braces.hcl
#    ^^^^^^ source.hcl meta.block.hcl meta.braces.hcl meta.mapping.key.hcl variable.other.readwrite.hcl
#          ^ source.hcl meta.block.hcl meta.braces.hcl
#           ^ source.hcl meta.block.hcl meta.braces.hcl keyword.operator.assignment.hcl
#            ^ source.hcl meta.block.hcl meta.braces.hcl
#             ^ source.hcl meta.block.hcl meta.braces.hcl string.quoted.double.hcl punctuation.definition.string.begin.hcl
#              ^^^^ source.hcl meta.block.hcl meta.braces.hcl string.quoted.double.hcl
#                  ^ source.hcl meta.block.hcl meta.braces.hcl string.quoted.double.hcl punctuation.definition.string.end.hcl
>  }
#^^ source.hcl meta.block.hcl meta.braces.hcl
#  ^ source.hcl meta.block.hcl meta.braces.hcl punctuation.section.braces.end.hcl
>}
#^ source.hcl meta.block.hcl punctuation.section.block.end.hcl
>